src/argilla/training/spacy.py

```diff
@@ -95,9 +100,10 @@ def __init__(
             self._dataset if isinstance(self._dataset, tuple) and len(self._dataset) > 1 else (self._dataset, None)
         )
         self._train_dataset_path = "./train.spacy"
-        self._eval_dataset_path = "./dev.spacy" if self._eval_dataset else None
+        self._eval_dataset_path = "./dev.spacy" if self._eval_dataset else "./train.spacy"
```

Comment:
do we need to provide an eval dataset by default? For me it feels weird to use the training dataset to do this.

Reply:
Yes, it is mandatory, the solution/workaround here is to use the train dataset, but note that here I just simplified the code, before we had the exact same thing just that we were setting it to None first and then setting it to `train` if it was None

But as it's mandatory I also agree with raising an error if not provided

Reply:
No, this seems like a better solution. Thanks